bash-notes.md

```diff
@@ -0,0 +1,43 @@
+# TASK-2: LINUX-UNIX-bandit-labs
+
+## Commands
+
+### ssh banditX@bandit.labs.overthewire.org -p 2220
```

Comment:
You can use following formating example for code block
```bash
$ ssh banditX@bandit.labs.overthewire.org -p 2220
```

Reply:
Finished editing.